catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java

```diff
@@ -82,11 +85,40 @@
 	private static final String VARIABLE_NAME = "test_variable";
 	private static final String TAG = null;
 	private Sprite sprite;
+	private Sprite secondSprite;
 
 	@Override
 	protected void setUp() throws Exception {
 		super.setUp();
 		sprite = new SingleSprite("testSprite");
+		secondSprite = new SingleSprite("secondSprite");
+	}
+
+	public void testSameSpriteReferenceAfterCloning() {
```

Comment:
Well, I think it doesn't specifically test the clone brick .. but this problem isn't only relevant to the clone brick (also to the PointToBrick - that's also why I've made an interface for these bricks). It tests whether the method refreshSpriteReferences() really does what it should do. It should set the correct reference to a sprite in these bricks (e.g. after shallow cloning when there is still the old sprite reference). 
That was my intention for this test ;)